### PR TITLE
Create 2024-03-20-dashboards.md

### PR DESCRIPTION
**Pull request by kevin-sumo**
Dependent PR: https://github.com/SumoLogic/sumologic-documentation/pull/3623

## Purpose of this pull request

This pull request is to create release note for Synced crosshairs and time zoom.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[crosshairs](https://sumologic.atlassian.net/browse/DASH-1619)

### CODE REVIEW

File: blog-service/2024-03-18-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+We're happy to announce two new dashboarding features:
+
+### Synced Crosshairs
+When hovering over a time series panel Sumo Logic now displays a related crosshair in all other time series panels within the dashboard. With the synced crosshairs you can quickly see how the data in one panel may relate to what you see within other panels.
+ 
+
+
+### Send Panel Zoom to Dashboard
+When zooming into a panel, Sumo Logic now provides an option to temporarily set the zoomed timerange as the dashboard time range. This allows you to quickly zoom all other panels of the dashboard to the same time range.
```
Comment (kimsauce) [inline blog-service/2024-03-18-dashboards.md:21]: Should it be _timerange_ or _time range_?

```suggestion
When zooming into a panel, Sumo Logic now provides an option to temporarily set the zoomed timerange as the dashboard time range. This allows you to quickly zoom all other panels of the dashboard to the same time range. 
```

Comment (JV0812) [reply blog-service/2024-03-18-dashboards.md]: Content updated.

File: blog-service/2024-03-20-dashboards.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - dashboards
+  - crosshair
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+We are excited to announce a new set of changes to enhance the readability within the dashboard panels. With these updates, you'll have more control over time ranges, making it easier to investigate specific periods of interest and analyze data effectively. Here's what's included in this update:
+
+- **Crosshair**. When you hover over a time series panel, Sumo Logic now displays a crosshair line with the time-series data. If other time series panels are also configured with overlapping time range, then when you hover over one time panel, crosshair will be displayed on the other panels as well. [Learn more](docs/dashboards/panels/).
```
Comment (jpipkin1) [inline blog-service/2024-03-20-dashboards.md:14]:
```suggestion
- **Crosshair**. When you hover over a time series panel, Sumo Logic now displays a crosshair line with the time-series data. If other time series panels are also configured with overlapping time range, then when you hover over one time panel, crosshair will be displayed on the other panels as well. [Learn more](/docs/dashboards/panels/).
```

File: blog-service/2024-03-20-dashboards.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - dashboards
+  - crosshair
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+We are excited to announce a new set of changes to enhance the readability within the dashboard panels. With these updates, you'll have more control over time ranges, making it easier to investigate specific periods of interest and analyze data effectively. Here's what's included in this update:
+
+- **Crosshair**. When you hover over a time series panel, Sumo Logic now displays a crosshair line with the time-series data. If other time series panels are also configured with overlapping time range, then when you hover over one time panel, crosshair will be displayed on the other panels as well. [Learn more](/docs/dashboards/panels/).
```
Comment (jpipkin1) [inline blog-service/2024-03-20-dashboards.md:14]:
```suggestion
- **Crosshair**. When you hover over a time series panel, Sumo Logic now displays a crosshair line with the time series data. If other time series panels are also configured with an overlapping time range, then when you hover over one time panel, a crosshair will be displayed on the other panels as well. [Learn more](/docs/dashboards/panels/).
```

**Review by jpipkin1 — APPROVED** [review]

File: blog-service/2024-03-18-dashboards.md (hunk 1 above)
Comment (kimsauce) [reply blog-service/2024-03-18-dashboards.md]: thank you!

**Pull request merged**